### PR TITLE
fix: change default f0 method from crepe to dio

### LINKED ISSUE #86: Do not use crepe by default  because of its performance

### PR DESCRIPTION
**Pull request by 34j**
Closes #86

BREAKING CHANGE: all default values for f0 method  changed

### CODE REVIEW

File: src/so_vits_svc_fork/utils.py [hunk 1]
```diff
@@ -243,15 +243,15 @@ def compute_f0(
...
+    method: Literal["crepe", "crepe-tiny", "parselmouth", "dio", "harvest"] = "dio",
     **kwargs,
 ):
     with timer() as t:
         wav_numpy = wav_numpy.astype(np.float32)
         wav_numpy /= np.quantile(np.abs(wav_numpy), 0.999)
         if method in ["dio", "harvest"]:
             f0 = compute_f0_pyworld(wav_numpy, p_len, sampling_rate, hop_length, method)
-        elif method == "crepe":
+        elif method == "dio":
```
Comment (GarrettConway) [inline src/so_vits_svc_fork/utils.py:254]: This seems like a typo, no?

Comment (34j) [reply src/so_vits_svc_fork/utils.py:254]: Nearly missed it!!!!

**Comment by 34j** [comment]
I think we should leave the defaults for inference as they were after all since not many files are processed.

**Comment by 34j** [comment]
I changed my mind, all the defaults will be dio

**Pull request merged**